jax/_src/lax/lax.py

```diff
@@ -1199,6 +1199,7 @@ def sort_key_val(keys: Array, values: ArrayLike, dimension: int = -1,
 
 def top_k(operand: ArrayLike, k: int) -> tuple[Array, Array]:
   """Returns top ``k`` values and their indices along the last axis of ``operand``.
+  Element order may not be preserved between ``operand`` and ``values``.
```

Comment:
I think we need a blank line before this.

Also, it may be more accurate to say "the order of returned values is implementation-dependent"